packages/fxa-auth-server/config/index.ts

```diff
@@ -1920,6 +1920,26 @@ const convictConf = convict({
       env: 'OTP_SIGNUP_DIGIT',
     },
   },
+  passwordForgotOtp: {
+    enabled: {
+      doc: 'Feature flag for init password reset with emailed OTP',
+      format: Boolean,
+      default: false,
+      env: 'OTP_PASSWORD_FORGOT_ENABLED',
+    },
+    digits: {
+      doc: 'Number of digits in token',
+      default: 8,
+      format: 'nat',
+      env: 'OTP_PASSWORD_FORGOT_DIGIT',
```

Comment:
should this be `DIGIT` singular or _`DIGITS`_ plural to match the `digits` on L1930?
(possibly moot since it looks like we have prior-art of `OTP_SIGNUP_DIGIT` singular above on L1920)